pallets/automation-time/rpc/src/lib.rs

```diff
@@ -122,6 +133,54 @@ where
 		})
 	}
 
+	fn query_fee_details(
+		&self,
+		encoded_xt: Bytes,
+		at: Option<Block::Hash>,
+	) -> RpcResult<FeeDetails<u64>> {
```

Comment:
The substrate pattern is to use `NumberOrHex` as the return type instead of `u64` (u64 is already being used in automation time). When NumberOrHex is used, the return values stop being human readable but there are existing substrate libs and patterns for parsing it. @chrisli30 any opinion on whether we should stick to the substrate pattern or use this for human readability?

Reply:
I think "human readability"’s importance is lower because developers are consumer of the call and can always find ways to translate the result.

I vote for better adaptability, so `NumberOrHex`?